root/static/scripts/guess-case/MB/GuessCase/Main.js

```diff
@@ -1,7 +1,6 @@
 /*
    This file is part of MusicBrainz, the open internet music database.
-   Copyright (c) 2005 Stefan Kestenholz (keschte)
-   Copyright (C) 2010 MetaBrainz Foundation
+   Copyright (C) 2016 MetaBrainz Foundation
```

Comment:
Append the year instead of replace it. E.g., "2010, 2016" or "2010–2016" (look at how it's done in other files). (Or just don't alter the copyright line at all. Wait and see what @bitmap or @gentlecat say.)